documentation/modules/about-hooks-for-migration-plans-api.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: CONCEPT
+[id="about-hooks-for-migration-plans-api_{context}"]
+= API-based hooks for {project-short} migration plans
+
+You can add hooks to a migration plan from the command line by using the {project-full} API.
+
+[discrete]
+== Default hook image
+The default hook image for an {project-short} hook is `quay.io/konveyor/hook-runner:latest`. The image is based on the Ansible Runner image with the addition of `python-openshift` to provide Ansible Kubernetes resources and a recent `oc` binary.
+// The most recent?
```

Comment:
I see that quay.io/konveyor/hook-runner keeps being updated but it's not that great that we refer to an image on quay.io (we can but we also need to explain how to change this image, for disconnected environments) and specifically to an image in `konveyor`

Reply:
@liranr23 can you please take a look at productizing the hook-runner image?

Reply:
let's use `registry.redhat.io/rhmtc/openshift-migration-hook-runner-rhel8:v1.8.2-2`